### ISSUE TITLE
DatePicker A11y: All content must reside within a WAI-ARIA landmark or labelled region role

### ISSUE THREAD

**Issue by rstrangfeld** [carbon-design-system/carbon#2503]
When using the DatePicker, DAP reports the violation 
"All content must reside within a WAI-ARIA landmark or labelled region role." It associates this violation with the first input element within the calendar, the numInput for the current year.

Looking in the DOM in browser tools, the application code puts a DatePicker within normal content inside a div having role="region", but the elements that make up the calendar appear within a div having class "flatpickr-calendar" immediately below the body, with no labelled role. 

Browser: Chrome 73.0.3683.86.
Carbon versions: carbon-components-react 6.71.0 and carbon-components 9.62.3
Offering/product: IBM MDM Express.

**Comment by nictownsend** [comment]
Reopening - https://codesandbox.io/s/codesandbox-o8wjv - DAP shows:

![image](https://user-images.githubusercontent.com/47386186/72814257-a7514380-3c5c-11ea-9884-ec188419602b.png)

**Comment by joshblack** [comment]
Hi there! :wave: That violation is one related to the sandbox itself. If you switch the `App` tags to `main` this violation will go away. Hope this helps!

**Comment by nictownsend** [comment]
Hi, the DatePicker is appending to the <body> element, not to the <div id="root"> where <App> is rendered. So changing <div id="root" role="main"> is not fixing it. Please could you explain how you'd change the sandbox to get it to pass?
